### PR TITLE
[Web3Wallet] Echo

### PR DESCRIPTION
**Pull request by alexander-lsvk**
## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

**Review by flypaper0 — APPROVED** [review]

File: Sources/Web3Wallet/Web3Wallet.swift [hunk 1]
```diff
@@ -35,10 +36,11 @@ public class Web3Wallet {
     /// - Parameters:
     ///   - metadata: App metadata
     ///   - signerFactory: Auth signers factory
-    static public func configure(metadata: AppMetadata, signerFactory: SignerFactory) {
+    static public func configure(metadata: AppMetadata, signerFactory: SignerFactory, echoHost: String = "echo.walletconnect.com", environment: APNSEnvironment) {
```
Comment (llbartekll) [inline Sources/Web3Wallet/Web3Wallet.swift:39]: should the echo host be an optional? not every project is using our PN solution.

**Review by llbartekll — APPROVED** [review]
looks good!

File: Sources/Web3Wallet/Web3Wallet.swift (hunk 1 above)
Comment (llbartekll) [reply Sources/Web3Wallet/Web3Wallet.swift:39]: add echo config? that is optional and contains both?

Comment (llbartekll) [reply Sources/Web3Wallet/Web3Wallet.swift:39]: or it's good, just add a default environment?

**Pull request merged**